cypress/integration/pages/topicPage/index.js

```diff
@@ -18,7 +19,11 @@ Object.keys(config)
         before(() => {
           Cypress.env('currentPath', currentPath);
 
-          const newPath = `${currentPath}${overrideRendererOnTest()}`;
+          // const newPath = `${currentPath}${overrideRendererOnTest()}`;
+          const overrideSuffix = overrideRendererOnTest();
+          const newPath = `${
+            currentPath + overrideSuffix + (overrideSuffix ? '&' : '?')
```

Comment:
is `overrideSuffix` potentially undefined? If so, because of the string interpolation then it could output a string with 'undefined' in it

Reply:
It is only ever '?renderer_env=live' or ''

Reply:
oh ok. thanks for clarifying

Reply:
no - either a string on empty string